components/TextEditor/TextEditorContainer.tsx

```diff
@@ -58,15 +68,15 @@ const TextEditorContainer: FunctionComponent<TextEditorContainerProps> = ({heigh
                 value={defaultText}
                 editorDidMount={handleIsEditorMounted}
                 options={TextEditorSettings}
-                
+
             />
         </div>
-            
-                <button className={scss.buttonBlack} onClick={submitForm}>Submit Code</button>
-         
+            <TokenLink href={link} as={as}>
+                <button className={scss.buttonBlack} type={"submit"} onSubmit={handleSubmit}>Submit Code</button>
+            </TokenLink>
```

Comment:
don't think we need the link - can we just keep it as the button?
Its also not actually a form. Could we keep it as just `onClick` like it was before?